content/docs/2.1/scalers/influxdb.md

```diff
@@ -0,0 +1,73 @@
++++
+title = "InfluxDB"
+layout = "scaler"
+availability = "v2.1+"
+maintainer = "Community"
+description = "Scale applications based on InfluxDB queries"
++++
+
+### Trigger Specification
+
+This specification describes the `InfluxDB` trigger that scales based on the results of a flux query.
+
+```yaml
+triggers:
+  - type: influxdb
+    metadata:
+      serverURL: http://influxdb:8086
+      organizationName: influx-org
+      organizationNameFromEnv: INFLUXDB_ORG_NAME # Optional: You can use this instead of `organizationName` parameter. See details in "Parameter List" section
+      thresholdValue: '4'
+      query: |
+        from(bucket: "bucket-of-interest")
+        |> range(start: -12h)
+        |> filter(fn: (r) => r._measurement == "stat")
+      authToken: some-auth-token
+      authTokenFromEnv: INFLUXDB_AUTH_TOKEN # Optional: You can use this instead of `authToken` parameter. See details in "Parameter List" section
+      metricName: influx-metric # Optional: This value will default to a masked version of the url if not set by the user (generated value would be `influxdb-influx-metric-influx-org`). If this value isn't set the generated value would be (`influxdb-https---xxx-influx_org`)
```

Comment:
```suggestion
      metricName: influx-metric # Optional: This value will default to a masked version of the url and organization name if not set by the user (metrics name value would be then `influxdb-https---xxx-influx_org`).
```

I'd rephrase this a little bit, the previous version was a little bit confusing. imho.